gradle.properties

```diff
@@ -1,10 +1,14 @@
 projectVersion=4.1.0-SNAPSHOT
-grailsVersion=6.0.0
-grailsGradlePluginVersion=6.0.0
-groovyVersion=3.0.11
-gebVersion=5.1
-seleniumVersion=4.1.4
-testcontainersVersion=1.20.2
+
+grailsVersion=6.2.2
```

Comment:
This will mean we only support the latest grails 6.x in 4.1.x, right?  I would assume we want to support all of the 6.x line, specifically 6.1?

Reply:
No, I would not think so. Is there any breaking changes between `6.0.x` and `6.2.x`?